### PR TITLE
Add production profile for logging

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- Added a production profile for logging

**Related issue(s)**
See also #7706

### CODE REVIEW

File: docs/logging/05-02-production-profile.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Logging production profile
+type: Configuration
+---
+## Overview
+
+To use production-ready Logging, you can install Kyma with the Logging production profile. Higher memory limits set for Loki and FluentBit logging solutions ensure stable log processing in a production environment.
```
Comment (a-thaler) [inline docs/logging/05-02-production-profile.md]: We should get more specific in the introduction, something like that:

It is a production-ready profile under restriction of using the out-of-the box solution.
Loki will not be high-available as it cannot scale out with the default persistence. In order to scale it out it requires an external storage for chunk and index store like GCS and Bigtable. 
Loki in the production profile has a retention period of 1 day and is meant to be used as a convenient way of accessing logs via the Kyma Console.

Comment (bszwarc) [reply docs/logging/05-02-production-profile.md]: 👍  I will add this.

**Comment by bszwarc** [comment]
We are putting this one back to in progress - to wait for the tests to finish. Once they are done, we will come up with a recommendation for our customers.

File: docs/logging/05-02-production-profile.md (hunk 1 above)
Comment (bszwarc) [reply docs/logging/05-02-production-profile.md]: Done.

File: docs/logging/05-01-logging.md [hunk 2]
```diff
@@ -29,5 +29,9 @@ This table lists the configurable parameters, their descriptions, and default va
 | **config.schema_configs.index.period** | Defines how long indexes and log chunks are retained. | `168h` |
 | **config.storage_config.boltdb.directory** | Specifies the physical location of indexes in `boltdb`. | `/data/loki/index` |
 | **config.storage_config.filesystem.directory** | Specifies the physical location of log chunks in `filesystem`. | `/data/loki/chunks` |
+| **loki.resources.limits.memory** | Maximum amount of memory available for Loki to use. | `300Mi` |
+| **loki.resources.limits.cpu** | Maximum amount of CPU available for Loki to use. | `200m` | 
+| **fluent-bit.resources.limits.memory** | Maximum amount of memory available for FluentBit to use. |`128Mi`|
```
Comment (kazydek) [inline docs/logging/05-01-logging.md:34]:
```suggestion
| **fluent-bit.resources.limits.memory** | Maximum amount of memory available for Fluent Bit to use. |`128Mi`|
```

File: docs/logging/05-01-logging.md [hunk 3]
```diff
@@ -29,5 +29,9 @@ This table lists the configurable parameters, their descriptions, and default va
...
+| **fluent-bit.resources.limits.cpu** | Maximum amount of CPU available for FluentBit to use. |`100m`|
```
Comment (kazydek) [inline docs/logging/05-01-logging.md:35]:
```suggestion
| **fluent-bit.resources.limits.cpu** | Maximum amount of CPU available for Fluent Bit to use. |`100m`|
```

File: docs/logging/05-02-production-profile.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+## Parameters
+
+The table shows the parameters used in the production profile and their values:
+
+ Parameter  | Description |  Value   | 
+|-----------|-------------|----------|
+| **loki.resources.limits.memory** | Maximum amount of memory available for Loki to use. | `512Mi` | 
+| **fluent-bit.resources.limits.memory** | Maximum amount of memory available for FluentBit to use. |`256Mi`| 
+
+## Use production profile
```
Comment (kazydek) [inline docs/logging/05-02-production-profile.md:20]:
```suggestion
## Use the production profile 
```

File: docs/logging/05-02-production-profile.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
+---
+title: Logging production profile
+type: Configuration
+---
+## Overview
```
Comment (kazydek) [inline docs/logging/05-02-production-profile.md:5]:
```suggestion

```

File: docs/logging/05-02-production-profile.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+>**NOTE:** This profile does not allow for horizontal scaling for which you need additional, dedicated storage systems.
+
+## Parameters
+
+The table shows the parameters used in the production profile and their values:
+
+ Parameter  | Description |  Value   | 
+|-----------|-------------|----------|
+| **loki.resources.limits.memory** | Maximum amount of memory available for Loki to use. | `512Mi` | 
+| **fluent-bit.resources.limits.memory** | Maximum amount of memory available for FluentBit to use. |`256Mi`|
```
Comment (kazydek) [inline docs/logging/05-02-production-profile.md:18]:
```suggestion
| **fluent-bit.resources.limits.memory** | Maximum amount of memory available for Fluent Bit to use. |`256Mi`| 
```

File: docs/logging/05-02-production-profile.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+<div tabs>
+  <details>
+  <summary>
+  Install Kyma with production-ready Logging
+ </summary>
+
+1. Create a Kubernetes cluster for Kyma installation.
+
+2. Apply an override that forces Logging to use the production profile:
```
Comment (kazydek) [inline docs/logging/05-02-production-profile.md:32]:
```suggestion
2. Apply an override that enforces the production profile for Logging:
```

File: docs/logging/05-02-production-profile.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+    fluent-bit.resources.limits.memory: "256Mi"
+  EOF
+  ```
+  </details>
+  <details>
+  <summary>
+  Enable configuration in a running cluster
+  </summary>
+
+1. Apply an override that forces Logging to use the production profile:
```
Comment (kazydek) [inline docs/logging/05-02-production-profile.md:57]:
```suggestion
1. Apply an override that enforces the production profile for Logging:
```

File: docs/logging/05-01-logging.md (hunk 2 above)
Comment (bszwarc) [reply docs/logging/05-01-logging.md:34]: 👍

File: docs/logging/05-01-logging.md (hunk 3 above)
Comment (bszwarc) [reply docs/logging/05-01-logging.md:35]: 👍

File: docs/logging/05-02-production-profile.md (hunk 5 above)
Comment (bszwarc) [reply docs/logging/05-02-production-profile.md:5]: 👍

File: docs/logging/05-02-production-profile.md (hunk 6 above)
Comment (bszwarc) [reply docs/logging/05-02-production-profile.md:18]: Done :)

File: docs/logging/05-02-production-profile.md (hunk 4 above)
Comment (bszwarc) [reply docs/logging/05-02-production-profile.md:20]: Done :)

File: docs/logging/05-02-production-profile.md (hunk 7 above)
Comment (bszwarc) [reply docs/logging/05-02-production-profile.md:32]: Done.

File: docs/logging/05-02-production-profile.md (hunk 8 above)
Comment (bszwarc) [reply docs/logging/05-02-production-profile.md:57]: Done.

**Pull request merged**